tests/metrics/network/dockerfile_image/README.md

```diff
@@ -0,0 +1,12 @@
+# Building the network image 
+
+```
+$ docker build -t cc-network .
+```
+
+# Verifying the network image
```

Comment:
Please use numbered steps. Also use the imperative:
Verify the network image with the following command: